### PR TITLE
Add playwright testing skeleton

### PR DESCRIPTION
**Pull request by dit7ya**
This PR adds basic CI support for running E2E frontend tests via [Playwright](https://playwright.dev/).

### CODE REVIEW

**Comment by dit7ya** [comment]
I think nix build is failing due to wrong `yarn.lock`.

File: .github/workflows/vast.yaml [hunk 1]
```diff
@@ -884,7 +897,6 @@ jobs:
       HOMEBREW_NO_ANALYTICS: 1
       HOMEBREW_NO_INSTALL_CLEANUP: 1
       HOMEBREW_NO_AUTO_UPDATE: 1
-      DESTDIR: "${{ github.workspace }}"
```
Comment (dominiklohmann) [inline .github/workflows/vast.yaml:887]: Note to the reviewer: Setting DESTDIR and CMAKE_INSTALL_PREFIX together breaks installations. This wasn't ever used to begin with.

File: plugins/web/CMakeLists.txt [hunk 2]
```diff
@@ -64,6 +64,8 @@ if (NOT VAST_ENABLE_WEB_UI)
   return()
 endif ()
 
+include(GNUInstallDirs)
```
Comment (dominiklohmann) [inline plugins/web/CMakeLists.txt:67]: Note to the reviewer: The installation was broken for non-bundled builds.

**Review by tobim — APPROVED** [review]
The cmake and workflow changes LGTM. Can't say anything about playwright itself.

**Pull request merged**